Server crashing

Thank you for this great tool

`/usr/src/app/openvpn.js:88
      const [pub, port] = props[this.clientProps.indexOf('Real Address')].split(':')
                                                                          ^

TypeError: Cannot read property 'split' of undefined
    at client.procData (/usr/src/app/openvpn.js:88:75)
    at /usr/src/app/openvpn.js:47:33
    at Array.forEach (<anonymous>)
    at Socket.<anonymous> (/usr/src/app/openvpn.js:47:13)
    at Socket.emit (events.js:311:20)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:209:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:186:23)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! openvpn-status@4.2.30 start: `node server.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the openvpn-status@4.2.30 start script.
`

--- a/openvpn.js
+++ b/openvpn.js
@@ -60,7 +60,6 @@ class client extends EventEmitter {
           this.procData(itm.toString());
         } catch (error) {
           console.log(`Could not process data item ${itm.toString()}`);
-          console.log(itm);
           console.log(error);
         }});
     })

--- a/openvpn.js
+++ b/openvpn.js
@@ -55,7 +55,14 @@ class client extends EventEmitter {
     this.connected = false
     this.socket.on('data', data => {
       const items = data.toString().split('\r\n').filter(itm => itm.length)
-      items.forEach(itm => this.procData(itm.toString()))
+      items.forEach(itm => {
+        try {
+          this.procData(itm.toString());
+        } catch (error) {
+          console.log(`Could not process data item ${itm.toString()}`);
+          console.log(itm);
+          console.log(error);
+        }});
     })
     this.socket.on('error', () => {
       this.connected = false
@@ -96,14 +103,19 @@ class client extends EventEmitter {
       }
     } else if (data.startsWith('ROUTING_TABLE') && this.state === STATE.status) {
       const props = data.split('\t').slice(1, data.length + 1)
-      const [pub, port] = props[this.clientProps.indexOf('Real Address')].split(':')
-      const client = Array.from(this.clients.values()).find((client) => client.pub === pub && client.port === port && this.netmask.contains(client.pub))
-      if (client) {
-        const vpnClient = this.clientProps.reduce((acc, prop, idx) => {
-          acc[prop] = prepProperty(props[idx])
-          return acc
-        }, {})
-        mkClient(vpnClient, client)
+      const realAddressIndex = this.clientProps.indexOf('Real Address');
+      if (realAddressIndex < props.length) {
+        const [pub, port] = props[realAddressIndex].split(':')
+        const client = Array.from(this.clients.values()).find((client) => client.pub === pub && client.port === port && this.netmask.contains(client.pub))
+        if (client) {
+          const vpnClient = this.clientProps.reduce((acc, prop, idx) => {
+            acc[prop] = prepProperty(props[idx])
+            return acc
+          }, {})
+          mkClient(vpnClient, client)
+        }
+      } else {
+         console.log(`props truncated ${data}`);
       }
     } else if (data.startsWith('END') && this.state === STATE.status) {
       this.oldClients.forEach((client, key) => {

